package.json

```diff
@@ -92,7 +92,7 @@
     "@notifee/react-native": "5.6.0",
     "@rainbow-me/react-native-animated-number": "0.0.2",
     "@rainbow-me/react-native-payments": "1.1.5",
-    "@rainbow-me/swaps": "0.1.17",
+    "@rainbow-me/swaps": "0.2.0",
```

Comment:
this includes changes to use `Signer` in the underlying lib:
https://github.com/rainbow-me/swaps/pull/19